Remove essentially all uses of "tree". (#201)
mlswg/mls-architecture: draft-ietf-mls-architecture.md

- Additional services may or may not be required depending on the application
  design:

>   - If assisted joining is desired (meaning that the group state is not

Reviewer: ```suggestion
  - If assisted joining is desired (meaning that the `ratchet_tree` extension is not
```

    provided in Welcome messages), there must be a method to download the

>     group state corresponding to a group.

Reviewer: ```suggestion
    ratchet tree corresponding to a group.
```

  - If assisted joining is desired and the Delivery Service is not able to

>     compute the group state itself (because some proposals or commits are sent

Reviewer: ```suggestion
    compute the group state itself, including the corresponding `ratchet_tree`
    object, (for example, because some proposals or commits are sent
```

    encrypted), there must be a method for group members to publish the updated

>     group state after each commit.

Reviewer: I think these cases all refer to the `ratchet_tree` extension, as opposed to the ratchet tree generally. I suggest leaving these as is.

- Group IDs, as decided by group creators and used to uniquely identify a group.

> - Associated with the per-user group state (specifically in

Reviewer 2: ```suggestion
- Application-level identifiers of public key material (specifically
```
